src/transaction/log_manager.c

```diff
@@ -13370,8 +13370,28 @@ cdc_find_user (THREAD_ENTRY * thread_p, LOG_PAGE * log_page, LOG_LSA process_lsa
   LOG_REC_SUPPLEMENT *supplement;
   char *data;
 
+  int tran_index = logtb_find_tran_index (thread_p, trid);
+  if (tran_index != NULL_TRAN_INDEX)
+    {
+      LOG_TDES *tdes = LOG_FIND_TDES (tran_index);
+
+      /* if transaction is active or aborted, it can not find the user information.
+       * because transaction user is logged right before commit record */
+
+      if (!LOG_ISTRAN_COMMITTED (tdes))
+	{
+	  /* TODO: set appropriate error and return */
```

Comment:
Sure, it is a refactoring factor for CDC.

Reply:
Do you have an issue with it? or why do you leave it as future work?

Reply:
OK, please make the issue if it doesn't exist yet. I can't find a related issue in Jira.

Reply:
Checking transaction status is removed! 

CDC thread can see the outdated LOG_TDES, because Transaction worker thread can modify the LOG_TDES. And it is not reasonable that cdc thread blocks the transaction worker from changing the LOG_TDES.